Sources/SourceKitLSP/Swift/CodeActions/ConvertStringConcatenationToStringInterpolation.swift

```diff
@@ -0,0 +1,138 @@
+//
+//  ConvertStringConcatenationToStringInterpolation.swift
+//  
+//
+//  Created by Lau Chun Kai on 3/7/2024.
+//
+
+import LanguageServerProtocol
+import SwiftRefactor
+import SwiftSyntax
+
+struct ConvertStringConcatenationToStringInterpolation: SyntaxRefactoringProvider {
+  static func refactor(syntax: ExprListSyntax, in context: Void) -> ExprListSyntax? {
```

Comment:
I would prefer if the refactoring operated primarily on `SequenceExprSyntax` instead of `ExprListSyntax`. Currently `ExprListSyntax` is only used inside `SequenceExprSyntax` but that might change in the future, while `SequenceExprSyntax` is the dedicated node for this kind of expression + operator list.

Reply:
I feel like I was bikeshedding between `SequenceExprSyntax` and `ExprListSyntax`, thanks for your explanation!